src/components/organisms/Carousel.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { ReactElement, TouchEvent, useEffect, useState } from 'react'
+import classNames from 'classnames/bind'
+import styles from './Carousel.module.css'
+
+const cx = classNames.bind(styles)
+
+export default function Carousel({
```

Comment:
question(blocking): Can you quickly explain why we are using a custom solution here? Are there no simple, zero-dependency carousel solutions available?

Reply:
There surely must be some, but it seemed a component small enough, not as critical and without too many required functionalities to justify the relatively small amount of code needed to go with a custom implementation and avoid adding a new dependency. 
Would you prefer me to replace it with a preexisting package?

Reply:
No, I am fine with your choice. I was just wondering if there was a quicker way, as you also mentioned bugs with this component. If everything is resolved and we do not need to invest more time, I completely agree with your choice.

Reply:
There is still a bug left to fix on the images loop when reaching the end of the list. The quick solution for the mvp would be to disable the loop and the "next" button when reaching the end.